liblll/CodeFragment.cpp

```diff
@@ -113,6 +113,22 @@ CodeFragment::CodeFragment(sp::utree const& _t, CompilerState& _s, bool _allowAS
 	}
 }
 
+namespace
+{
+bool validFunctionalInstruction(string us)
+{
+	auto it = c_instructions.find(us);
+	if (
```

Comment:
I'm not sure what's more readable, but in general if you are returning bool you don't need to wrap bool expression with `if`.